src/angular-cesium/models/vec3.ts

```diff
@@ -0,0 +1,5 @@
+export interface Vec3 {
```

Comment:
what the difference between vec3 and cartesian3? 
cant we use cartesian3?

Reply:
its not the same. cartesian3 must have clone() + equals()